experiments/spotify/1_no_code_consume_source/README.md

```diff
@@ -0,0 +1,36 @@
+# Consume a ready source without Python code
+We can load a source from the command line. Imagine this:
+
+```
+pip install python-dlt
+pip install python-dlt-contrib
+
+dlt init
+
+vi .dlt/secrets
+
+dlt source info dlt.contrib.sources.spotify
+
+> displays the source docstring and parameters to be passed - all from function signature!
+
+dlt load --to bigquery --dataset spotify dlt.contrib.sources.spotify --data-dir my_data --tables library,history
+```
+
+## How does it work
+
+1. `dlt init` created default pipeline folder structure with empty `secrets` and `config` where options are present but commented out. `dlt init` can take a parameter which is a name of the workflow ie. `consume` to consume source, `pipeline` to write a new pipeline, `source` to develop source
```

Comment:
what does it mean to consum a source vs. write a new pipeline vs. develop a source? We need to better define these things and align on them. Based on the conversation this morning, I think we have different definitions here

Reply:
let's build a glossary more because you clearly think about pipelines differently. IMO pipeline can have many sources with different schemas (you call them manifests) and set of python transformations (ie to clean up the data in python). in general I see the pipeline not as blackbox but as an api where I have control over whats going on inside, not just input and output